logstash-core/lib/logstash/config/source/local.rb

```diff
@@ -102,7 +102,7 @@ def get_unmatched_files
       end
 
       def valid_encoding?(content)
-        content.ascii_only? && content.valid_encoding?
+        content.ascii_only? || content.valid_encoding?
```

Comment:
@jsvd I don't think this interferes with the escape feature, my logic is:

```rb
    def process(value)
      case value[1]
      when '"', "'", "\\"
        value[1]
      when "n"
        "\n"
      when "r"
        "\r"
      when "t"
        "\t"
      else
        value
      end
    end
```

none of these are different between UTF8 and ASCII => we're good?